src/views/ReviewConfirm.vue

```diff
@@ -106,7 +113,7 @@ export default class ReviewConfirm extends Mixins() {
   }
 
   /** Called when component is mounted. */
-  mounted (): void {
+  async mounted (): Promise<void> {
```

Comment:
Not async?

Reply:
Good catch, not anymore. 
at one point was making the profile request in this hook until i found it elsewhere.
Removed!